PyInstaller/utils/hooks/conda.py

```diff
@@ -0,0 +1,196 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+# language=rst
+"""
+Additional helper methods for working specifically with Anaconda distributions
+are found at :attr:`PyInstaller.util.hooks.conda_support`. These functions find
+and parse the distribution metadata from json files located in the
+``conda-meta`` directory.
+
+.. versionadded:: 4.1.0
+
+This module is available only if run inside a Conda environment. Usage of this
+module should therefore be guarded by a try/except clause::
+
+    try:
+        from PyInstaller.utils.hooks import conda_support
+    except ImportError:
+        # Not using conda.
+        pass
+    else:
+        # Code goes here. e.g.
+        binaries = conda_support.collect_dynamic_libs("numpy")
```

Comment:
They're not quite equivalent. If a conda user decides they'd rather use `venv` (or something similar) then this would break in that environment because `is_conda` would be true, but the environment wouldn't have the `conda-metda` dir. But I was going to use @rokm's suggestion of adding a `compat.is_pure_conda` which would allow for a plain `if` statement instead of a `try: expect`.